src/Microsoft.PowerShell.Commands.Utility/resources/UtilityCommonStrings.resx

```diff
@@ -174,9 +174,6 @@
   <data name="TypeNotSupported" xml:space="preserve">
     <value>'{0}' is not supported in this system.</value>
```

Comment:
Would `'{0}' is not supported on this platform.` be better (to me it feels more consistent in terms of language compared to similar errors in .Net Core).

Reply:
@bergmeister It is out the PR. Please push new PR or open new tracking issue.